dir_class.py

```diff
@@ -94,3 +94,12 @@ def update_tags(self, album_artist, album_title, f):
         audio["album"] = album_title
         audio["albumartist"] = album_artist
         audio.save()
+
+    def edit(self):
+        self.data_about_files()
+
+    def data_about_files(self):
+        for x, y, z in self.base_dir:
```

Comment:
`self.base_dir` to ścieżka do katalogu (string), a nie sam katalog. Brakuje `listdir` albo `walk`. Podejrzyj jak korzystałaś z tych funkcji wcześniej.